modules/apps/foundation/frontend-theme/responsive-boat-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,278 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+
+
+/********** VARIABLE SECTION **********/
+$expandTime: 0.4s; 				// time it takes for mobile navbar to expand or collapse
```

Comment:
Since this is relating to mobile, I would rename this variable to be a little more descriptive about what it does rather than relying on the comments for further explanation.